backend/prisma/seed.ts

```diff
@@ -0,0 +1,49 @@
+import { PrismaClient } from '@prisma/client';
+import * as fs from 'fs';
+
+const prisma = new PrismaClient();
+
+const csvText = './prisma/csv/texts.csv';
+const csvConfig = './prisma/csv/configs.csv';
+
+async function seedData(): Promise<void> {
+    try {
+        const [textContents, configContents] = await Promise.all([
+            fs.promises.readFile(csvText, 'utf-8'),
+            fs.promises.readFile(csvConfig, 'utf-8')
+        ]);
+
+        const textlines = textContents.trim().split('\n').slice(1);
+        const configlines = configContents.trim().split('\n').slice(1);
+
+        for (const line of textlines) {
+            const [key, en, fr] = line.split(',');
+            const existingText = await prisma.texts.findFirst({ where: { key } });
+            if (!existingText) {
+                await prisma.texts.create({
+                    data: { key, en, fr }
+                });
+            }
+
+        }
+        console.log('✅  All Data from ' + csvText + ' seeded successfully');
+
+        for (const line of configlines) {
+            const [key, value] = line.split(',');
+            const existingConfig = await prisma.texts.findFirst({ where: { key } });
```

Comment:
- Typo: You should check for the key in the configs table not the texts table.
- Suggestion: It would be better to use findUnique() instead of findFirst(), so the key in the table should be marked @unique.
- Breaking the DRY principle: The same for loop is writting twice to work on texts and configs. It would better to write a function for it.